Catappult/appcoins-wallet-android
# [feature/APPC-2283] Show fiat on transactions details

**What does this PR do?**
  
  Shows fiat on transaction details, both paid fiat information and converted fiat information when the currency from the global balance is different from the currency paid on the transaction. Also changes done in the layout to replicate the design.


**Database changed?**
  
No

**Where should the reviewer start?**
- TransactionDetailActivity.java
- TransactionDetailViewModel.java
- LocalCurrencyConversionService.kt 

**How should this be manually tested?**

open a transaction without fiat values and check value
open a transaction with fiat and check values.
open a transaction paid with a different currency from the balance currency

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-2283

**Code Review Checklist**
